interop/script/run.exs

```diff
@@ -13,49 +13,39 @@ Logger.configure(level: level)
 Logger.info("Rounds: #{rounds}; concurrency: #{concurrency}; port: #{port}")
 
 alias Interop.Client
+alias GRPC.Client.Adapters.{Mint, Gun}
 
 {:ok, _pid, port} = GRPC.Server.start_endpoint(Interop.Endpoint, port)
 
-1..concurrency
-|> Task.async_stream(fn _cli ->
-  ch = Client.connect("127.0.0.1", port, interceptors: [GRPCPrometheus.ClientInterceptor, GRPC.Logger.Client])
-
-  for _ <- 1..rounds do
-    Client.empty_unary!(ch)
-    Client.cacheable_unary!(ch)
-    Client.large_unary!(ch)
-    Client.large_unary2!(ch)
-    Client.client_compressed_unary!(ch)
-    Client.server_compressed_unary!(ch)
-    Client.client_streaming!(ch)
-    Client.client_compressed_streaming!(ch)
-    Client.server_streaming!(ch)
-    Client.server_compressed_streaming!(ch)
-    Client.ping_pong!(ch)
-    Client.empty_stream!(ch)
-    Client.custom_metadata!(ch)
-    Client.status_code_and_message!(ch)
-    Client.unimplemented_service!(ch)
-    Client.cancel_after_begin!(ch)
-    Client.cancel_after_first_response!(ch)
-    Client.timeout_on_sleeping_server!(ch)
-  end
-  :ok
-end, max_concurrency: concurrency, ordered: false, timeout: :infinity)
-|> Enum.to_list()
-
-# defmodule Helper do
-#   def flush() do
-#     receive do
-#      msg ->
-#        IO.inspect(msg)
-#        flush()
-#     after
-#      0 -> :ok
-#     end
-#   end
-# end
-# Helper.flush()
+for adapter <- [Gun, Mint] do
+  1..concurrency
+  |> Task.async_stream(fn _cli ->
+                          ch = Client.connect("127.0.0.1", port, interceptors: [GRPCPrometheus.ClientInterceptor, GRPC.Logger.Client], adapter: adapter)
+
+                          for _ <- 1..rounds do
+                            Client.empty_unary!(ch)
+                            Client.cacheable_unary!(ch)
+                            Client.large_unary!(ch)
+                            Client.large_unary2!(ch)
+                            Client.client_compressed_unary!(ch)
+                            Client.server_compressed_unary!(ch)
+                            Client.client_streaming!(ch)
+                            Client.client_compressed_streaming!(ch)
+                            Client.server_streaming!(ch)
+                            Client.server_compressed_streaming!(ch)
+                            Client.ping_pong!(ch)
+                            Client.empty_stream!(ch)
+                            Client.custom_metadata!(ch)
+                            Client.status_code_and_message!(ch)
+                            Client.unimplemented_service!(ch)
+                            Client.cancel_after_begin!(ch)
+                            Client.cancel_after_first_response!(ch)
+                            Client.timeout_on_sleeping_server!(ch)
+                          end
+                          :ok
+  end, max_concurrency: concurrency, ordered: false, timeout: :infinity)
```

Comment:
How about extracting the anonymous function to a variable?

Reply:
https://github.com/elixir-grpc/grpc/pull/272/commits/d3e026f6bbdb462d044247f94a5a1104866e1b23